bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -5,7 +5,13 @@
 {% extends "firefox/base-resp.html" %}
 
 {% block page_title_prefix %}{% endblock %}
-{% block page_title %}{{ _('Firefox — Desktop browser, Android, OS, Hello, Sync, Marketplace') }}{% endblock %}
+{% block page_title %}
+{% if l10n_has_tag('firefox_ios') and waffle.switch('firefox-ios-global') %}
```

Comment:
We can't use the `firefox_ios` tag because that tag was introduced months ago. We can use the `tracking_protection` tag already used below, or change that to `firefox_v1_1` for both. Your call.